model/task.go

```diff
@@ -130,6 +130,18 @@ func (t *Task) OwnedBy(address common.Address) bool {
 	return strings.EqualFold(t.Owner, address.Hex())
 }
 
+// A task is runable when both of these condition are matched
+//  1. Its max execution has not reached
+//  2. Its expiration time has not reached
+func (t *Task) Runable() bool {
```

Comment:
I’m not sure about the naming in go, but to be consistent with the name `TaskIsNotRunable`, I’d call this function `isRunnable()`. The `is` will clearly indicate that this is boolean returning function.

Reply:
Yeah, understood and sounds good!